packages/nextjs-mf/src/plugins/container/InvertedContainerPlugin.ts

```diff
@@ -228,11 +232,11 @@ class InvertedContainerPlugin {
 
             return Template.asString([
               '',
-              'var currentChunkId = "__INSERT_CH_ID__MF__";',
+              'var currentChunkId = __INSERT_CH_ID__MF__;',
               `if(currentChunkId) {`,
               Template.indent([
-                `__webpack_require__.getEagerSharedForChunkId(currentChunkId,__webpack_require__.initRemotes);`,
-                `__webpack_require__.getEagerRemotesForChunkId(currentChunkId,__webpack_require__.initConsumes)`,
+                `if(__webpack_require__.getEagerSharedForChunkId) {__webpack_require__.getEagerSharedForChunkId(currentChunkId,__webpack_require__.initRemotes)}`,
```

Comment:
intRemotes and initConsumes appear to be flipped here?
getShared is using initRemotes and getRemotes initConsumes